src/routes/Onboarding/BrowserOnboardingRoutes.tsx

```diff
@@ -0,0 +1,498 @@
+          to edit their parameters.
+        </p>
+        <OnboardingButtons
+          currentSlug={thisOnboardingStatus}
+          platform="browser"
+        />
+      </OnboardingCard>
+    </div>
+  )
+}
+
+function PromptToEdit() {
+  const thisOnboardingStatus: BrowserOnboardingPath = '/browser/prompt-to-edit'
+
+  // Click the text-to-cad dropdown button if it's available
+  useEffect(() => {
+    const dropdownButton = document.querySelector(
+      `[data-${ONBOARDING_DATA_ATTRIBUTE}="ai-dropdown-button"]`
+    )
+    if (dropdownButton === null) {
+      console.error(
+        `Expected dropdown is not present in onboarding step '${thisOnboardingStatus}'`
+      )
+      return
+    }
+
+    if (dropdownButton instanceof HTMLButtonElement) {
+      dropdownButton.click()
+    }
+  }, [])
+  // Close the panes on mount, close on unmount
+  useOnboardingPanes()
+
+  // Make it so submitting the command just advances the onboarding
+  useAdvanceOnboardingOnFormSubmit(thisOnboardingStatus)
+
+  return (
+    <div className="fixed inset-0 z-50 grid items-center justify-center p-16">
+      <OnboardingCard className="col-start-3 col-span-2">
+        <h1 className="text-xl font-bold">Modify with Zoo Text-to-CAD</h1>
+        <p className="my-4">
+          Text-to-CAD not only can <strong>create</strong> a part, but also{' '}
+          <strong>modify</strong> an existing part. In the dropdown, you’ll see
+          “Modify with Zoo Text-to-CAD”. Once clicked, you’ll describe the
+          change you want for your part, and our AI will generate the change.
+          Once again, this will cost <strong>one credit per minute</strong> it
+          took to generate. Once again, most of the time, this is under a
+          minute.
+        </p>
+        <OnboardingButtons
+          currentSlug={thisOnboardingStatus}
+          platform="browser"
+        />
+      </OnboardingCard>
+    </div>
+  )
+}
+
+function PromptToEditPrompt() {
+  const thisOnboardingStatus: BrowserOnboardingPath =
+    '/browser/prompt-to-edit-prompt'
+  const prompt =
+    'Change the housing to be for a 150 mm diameter fan, make it 30 mm tall, and change the color to purple.'
+
+  // Ensure panes are closed
+  useOnboardingPanes()
+
+  // Enter the prompt-to-edit flow with a prebaked prompt
+  const [isReady, setIsReady] = useState(
+    isModelingCmdGroupReady(commandBarActor.getSnapshot())
+  )
+  useOnModelingCmdGroupReadyOnce(() => {
+    setIsReady(true)
+    commandBarActor.send({
+      type: 'Find and select command',
+      data: {
+        groupId: 'modeling',
+        name: 'Prompt-to-edit',
+        argDefaultValues: {
+          selection: {
+            graphSelections: [],
+            otherSelections: [],
+          } satisfies Selections,
+          prompt,
+        },
+      },
+    })
+  }, [])
+
+  // Make it so submitting the command just advances the onboarding
+  useAdvanceOnboardingOnFormSubmit(thisOnboardingStatus)
```

Comment:
This is probably my least-favorite block of code in this whole thing. The prompt-to-edit command is not available in the command palette until there is an engine connection, so we have to wait for that to find and select it.

Then, the command palette steals focus away from the onboarding modal, so we have to override form submission in case the user hits <kbd>Enter</kbd> while command palette has focus, so that there's no way they can accidentally submit a prompt and burn a credit.